source/payment-service-provider.html.erb

```diff
@@ -43,8 +43,9 @@ title: GOV.UK Pay’s Payment Service Provider
     </div>
     <div class="column-two-thirds">
         <h1 class="heading-large">GOV.UK Pay’s Payment Service Provider</h1>
-        <p>GOV.UK Pay takes payment, but uses a Payment Service Provider (PSP) to process the payment. GOV.UK Pay has procured a PSP for you to use. This means you don’t need a separate contract with a PSP to use GOV.UK Pay.</p>
-
+        <p>GOV.UK Pay takes payment, but uses a Payment Service Provider (PSP) to process it. GOV.UK Pay has procured a PSP for you to use. This means you don’t need a separate contract with a PSP to use GOV.UK Pay.</p>
+        <p>If you already have your own PSP, GOV.UK Pay supports Worldpay Corporate Gateway, Barclays ePDQ and Barclays Smartpay/Adyen.</p>
```

Comment:
Might be a good idea to include the new names Barclaycard came up with when they had rummage around in the Scrabble letter tiles bag: Smartpay I for ePDQ and Smartpay A for the other one.

Reply:
the original content is how we refer to the PSPs throughout the product pages and docs. @tillwirth you probably have the best idea of how services refer to epdq and smartpay - what do you think - should we include the latest brand names?